pages/about_us.md

```diff
@@ -0,0 +1,7 @@
+---
+title: About Us
+keywords:
+---
+
```

Comment:
Elixir - Elixir BE info (https://rdm.elixir-belgium.org/elixir) should be moved to this "About us" page. But we can do it later.